src/fillTank.test.js

```diff
@@ -1,11 +1,56 @@
 'use strict';
 
+const { fillTank } = require('./fillTank');
+
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  let customer;
+
+  beforeEach(() => {
+    customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+  });
 
-  it('should ', () => {
+  it('should fill the tank with the full amount'
+    + 'if the amount is not given', () => {
+    fillTank(customer, 59);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+    expect(customer.money).toBe(1112);
+  });
+
+  it('should pour only what will fit if the amount is'
+    + 'greater than the tank can accommodate', () => {
+    fillTank(customer, 59, 50);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+    expect(customer.money).toBe(1112);
+  });
 
+  it('should fill only what the client can afford', () => {
```

Comment:
explain what in test name what does it mean "client can afford"

Reply:
transfer costs, missed, fixed)